plantcv/plantcv/print_results.py

```diff
@@ -1,18 +1,169 @@
 # Print Numerical Data
 
+from plantcv.plantcv import outputs
 
-def print_results(filename, header, data):
+
+def print_results(filename):
     """Print result table
 
     Inputs:
     filename = filename
-    header   = result data table headers
-    data     = result data table values
 
     :param filename: str
-    :param header: list
-    :param data: list
     :return:
     """
-    print('\t'.join(map(str, header)))
-    print('\t'.join(map(str, data)))
+    result = open(filename, "a")
+    if 'bound_horizontal' in outputs.measurements:
+        bound_header = [
+            'HEADER_BOUNDARY_H',
```

Comment:
`plantcv-pipeline.py` is expecting the header keyword to be `HEADER_BOUNDARY_#` where # is the boundary line position. I think we will need to refactor that in `plantcv-pipeline.py`. I'm thinking we should add the boundary line position as an output "measurement" for these functions. For the horizontal we store it as `y-position`.

Reply:
Looking at `plantcv-pipeline.py` I'm a little confused what is happening with the `analyze_bound*` data. It seems like it's only storing data from the horizontal function upon first look? Also, should I change the way landmark data is returned to match the header list, data list return of the other functions?

Reply:
Yep, we will need to update `plantcv-pipeline.py` to accommodate somewhat different `analyze_bound_horizontal` output and to add processing for `analyze_bound_vertical`. And you are right about changing the output of the landmark functions to match the other `analyze_*` functions.